src/pages/projects/ProjectLandingPage.tsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Accordion, AccordionSummary, Typography } from '@material-ui/core';
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
+import { useTranslation } from 'react-i18next';
+import { PageHeader } from '../../components/PageHeader';
+import { CristinProject } from '../../types/project.types';
+import {
+  getProjectName,
+  getProjectManagerName,
+  getProjectPeriod,
+} from '../registration/description_tab/projects_field/projectHelpers';
+
+const StyledGeneralInfoBox = styled.div`
+  margin-bottom: 1rem;
+  display: grid;
```

Comment:
anbefaler å teste ut materia-ui sin grid. har begynt å like den :-)

Reply:
husker vi snakket om det for leeeenge siden, men husker ikke hvorfor vi droppa det... kan lese litt på det igjen 👍